view/theme/frio/style.php

```diff
@@ -208,8 +208,8 @@
 	'$background_repeat'           => $background_repeat,
 	'$login_bg_image'              => $login_bg_image,
 	'$login_bg_color'              => $login_bg_color,
-	'$font_color_darker'           => $font_color_darker ?? '#222',
-	'$font_color'                  => $font_color ?? '#444',
+	'$font_color_darker'           => $font_color_darker ?? '#000',
+	'$font_color'                  => $font_color ?? '#111',
```

Comment:
Please separate these changes in another PR to allow for discussion.

Reply:
Crap i thought I had unchanged that since someone else was already working on it.

Reply:
Is there are way to just straight up reject this change? It was an accident.

Reply:
Sure, you can do an interactive rebase onto `develop` and drop the c78e2345a09fc99660c2564778442d2929ccbf9d commit from the branch.